guides/source/action_text_overview.md

```diff
@@ -97,6 +97,8 @@ Then refer to this field in the form for the model:
 <% end %>
 ```
 
+**Note:** The `rich_text_area` form helper automatically adds the `trix-content` CSS class to the Trix editor element. `trix-content` is responsible for the editor's [default styling](https://github.com/rails/rails/blob/adc5eb66f861a98da32f7c682155360d5b9a4a52/actiontext/app/helpers/action_text/tag_helper.rb#L14). Passing a `class` argument overrides `trix-content`, removing the default styles.
```

Comment:
I think it might be better to rely on the API docs to convey this information instead.  The user should already be checking the API docs to see if a `:class` option is supported, since there is no indication of it otherwise.  Though we could make the information more easily accessible with a link:

````diff
diff --git a/guides/source/action_text_overview.md b/guides/source/action_text_overview.md
index 34d1deba6e..922161cc9d 100644
--- a/guides/source/action_text_overview.md
+++ b/guides/source/action_text_overview.md
@@ -85,7 +85,7 @@ end
 
 **Note:** you don't need to add a `content` field to your `messages` table.
 
-Then refer to this field in the form for the model:
+Then use [`rich_text_area`] to refer to this field in the form for the model:
 
 ```erb
 <%# app/views/messages/_form.html.erb %>
@@ -114,6 +114,8 @@ class MessagesController < ApplicationController
 end
 ```
 
+[`rich_text_area`]: https://api.rubyonrails.org/classes/ActionView/Helpers/FormHelper.html#method-i-rich_text_area
+
 ## Rendering Rich Text content
 
 Action Text will sanitize and render rich content on your behalf.
````

Reply:
@stevepolitodesign I noticed you marked this conversation as resolved, but did not apply the change.  Was that intentional?  Do you have any concerns?

Reply:
@jonathanhefner sorry about that, I'm still a PR n00b. I just made those changes. 

Thank you for your patience on this!

Reply:
@stevepolitodesign No worries!  To clarify, though, I meant add the API docs link **instead of** the above paragraph.  Generally, the guides provide a high-level overview, while the API docs delve into details such as default option values.  Trying to include too many details in the guides would eventually make them unwieldy (both to read and maintain), and would duplicate a lot of the work of the API docs.

Also, when you're satisfied with all the changes, would you mind [squashing the commits](https://guides.rubyonrails.org/contributing_to_ruby_on_rails.html#squashing-commits)?

Reply:
@jonathanhefner thank you for the clarification. I went ahead and removed the paragraph per your suggestion. If this looks good I can squash these down.